imap_processing/idex/idex_l2b.py

```diff
@@ -0,0 +1,144 @@
+"""
+Perform IDEX L2b Processing.
+
+Examples
+--------
+.. code-block:: python
```

Comment:
Nice work formatting this properly. It is something I wish our team (myself included) did more of.